perllib/FixMyStreet/Script/Reports.pm

```diff
@@ -28,7 +28,10 @@ sub send(;$) {
     my $base_url = FixMyStreet->config('BASE_URL');
     my $site = $site_override || CronFns::site($base_url);
 
-    my $states = [ 'confirmed', 'fixed' ];
+    my $states = [
+        FixMyStreet::DB::Result::Problem::open_states(),
+        'fixed'
```

Comment:
Can probably drop fixed in fact, given this matches nothing, and all fixeds are meant to be user/council.